test/server/getters/get-profit-loss.js

```diff
@@ -89,6 +98,81 @@ describe("server/getters/get-profit-loss#bucketRangeByInterval", () => {
   });
 });
 
+
+describe("tests for test/profitloss.db", () => {
+  var connection = null;
+  var augur = new Augur();
+
+
+  beforeEach((done) => {
+    sqlite3.verbose();
+    connection = Knex({
+      client: "sqlite3",
+      connection: {
+        filename: "./test/profitloss.db",
+      },
+      acquireConnectionTimeout: 5 * 60 * 1000,
+      useNullAsDefault: true,
+      postProcessResponse: postProcessDatabaseResults,
+    });
+
+    done();
+  });
+
+  it("has 2 trades", (done) => {
+    connection("trades").select("*").asCallback((err, results) => {
+      assert.isNull(err);
+      assert.equal(results.length, 2);
+      done();
+    });
+  });
+
+  const universe = "0x1b8dae4f281a437e797f6213c6564926a04d9959";
+  const account1 = "0x913da4198e6be1d5f5e4a40d0667f70c0b5430eb";
+  const account2 = "0xbd355a7e5a7adb23b51f54027e624bfe0e238df6";
+  const endTime = Date.now();
+  it("has a total PL of 4eth for account1", (done) => {
+    getProfitLoss(connection, augur, universe, account1, 0, endTime, endTime, (err, results) => {
+      try {
+        assert.deepEqual(results, [{
+          "lastPrice": "0.1",
+          "profitLoss": {
+            "meanOpenPrice": "0",
+            "position": "0",
+            "realized": "4",
+            "total": "4",
+            "unrealized": "0",
+          },
+          "timestamp": endTime,
+        }]);
+      } catch(e) {
```

Comment:
Hm, why use `try { ... } catch (e) { done(e) }` instead of just allowing the `assert.deepEqual` to throw?

Reply:
Because it throws from an async call which doesnt catch and call done